Python/optimizer.c

```diff
@@ -836,6 +836,12 @@ translate_bytecode_to_trace(
 
                         if (uop == _PUSH_FRAME) {
                             assert(i + 1 == nuops);
+                            if (opcode == FOR_ITER_GEN || BINARY_SUBSCR_GETITEM) {
```

Comment:
Condition is always true:

```suggestion
                            if (opcode == FOR_ITER_GEN || opcode == BINARY_SUBSCR_GETITEM) {
```